app/decorators/decidim/users/admin/users_controller_decorator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#
+# This decorator adds the capability to the controller to query users
+# filtering by User role `department_admin`.
+#
+Decidim::Admin::UsersController.class_eval do
+  alias_method :original_method, :collection
+
+  private
+
+  def collection
+    if current_user.department_admin?
+      @collection ||= current_organization.users_with_any_role.where("'department_admin' = ANY(roles)")
```

Comment:
We should also filter by the area of the current_user

Reply:
Is not necessary because area is not displayed in the form

Reply:
the area is the department, i think it is displayed.

On the other hand, I was asking to filter by area. A department_admin user, should ony see other department_admin users only from the same area/department. Thus, we must filter by area, isn't it?

Reply:
I think the area is not shown when creating a new admin

![Captura de pantalla de 2021-04-27 08-46-50](https://user-images.githubusercontent.com/75725233/116197672-4fde3500-a735-11eb-82f4-ae22c939a70f.png)

An admin deparment can only add users to the same department (I think it's called a role) and on the other hand, the new department admin will have the same area as the creator.

Reply:
That's wrong or there is a bug, it should render like that:
![image](https://user-images.githubusercontent.com/199462/116368640-da439900-a808-11eb-9d62-7370d5b431a1.png)